_projects/community-share.md

```diff
@@ -4,7 +4,7 @@ date: 2015-06-01
 link: https://github.com/communityshare/communityshare
 intro: Come help with
 image: /assets/images/communityshare.png
-lead: Josh Schachter & Ben Reynwar
```

Comment:
Yep, he's still the product owner. I'll add Kristin as well since she's the product manager now.